TESTING.md

```diff
@@ -148,68 +148,84 @@ Running tests against an iOS device locally is not currently supported.
 
 ### React-native
 
-The React-native tests drive real, remote mobile devices using BrowserStack. As a Bugsnag employee you can access the necessary credentials in our shared password manager.
+The React-native tests drive real, remote mobile devices using BrowserStack. As a Bugsnag employee you can access the 
+necessary credentials in our shared password manager.
 
-In addition, the react-native test fixture requires the current version of the repo to be published to a locally owned NPM repository (! Not the official NPMJS repository !). This can be locally or remotely hosted, but should be versioned appropriately.  Three bits of information will need to be passed into the test run as environment variables in order to access this package:
+The test fixture (a React Native app) that tests are run against needs to be built with a published version of 
+@bugsnag/react-native.  By default, the build process will base this on the current branch/comment, 
+e.g. `7.2.0-my-branch.231f6ef7`.  This can be overridden using the environment variable NOTIFIER_VERSION, which is useful during development when 
+making test, but not notifier, changes.
 
+If building against the current branch/commit, the packages must be published to a locally owned NPM repository 
+(! Not the official NPMJS repository !). This can be locally or remotely hosted, but should be versioned appropriately.  
+
+Three bits of information will need to be passed into the test run as environment variables in order to 
+access this package:
 - `REG_BASIC_CREDENTIAL`: the basic auth credentials of an account able to access the repository
 - `REG_NPM_EMAIL`: the email of the user accessing the repository
 - `REGISTRY_URL`: the remote address of the repository
 
-The targeted release of `@bugsnag/react-native` must be tagged with the short hash of the current commit in order to be picked up by the gradle build process.
+The targeted release of `@bugsnag/react-native` must be tagged with the short hash of the current commit in order to be 
+picked up by the gradle build process.
 
-There may be several react-native versions that can be targeted.  
-These should be set to the `REACT_NATIVE_VERSION` environment variable according to the table below:
+There may be several react-native versions that can be targeted and the `REACT_NATIVE_VERSION` environment variable 
```

Comment:
May be? I think there just are (and always will be) several versions. Unless I have misunderstood what "may" is doing in this sentence.

Reply:
I was just being pedantic over whether 2 constituted "many" - have now changed to "are".